core/arch/arm/kernel/boot.c

```diff
@@ -188,6 +188,29 @@ static void init_vfp_nsec(void)
 }
 #endif
 
+/*
+ * Check for supported Crypto Extensions (ARMv8 aarch32/aarch64)
+ * In case one of instructions is not supported false is returned.
+ */
+static bool check_cpuid_ce(void)
+{
+	uint32_t isar5 = read_isar5();
+
+	if (!(isar5 | ID_ISAR5_AES))
+		return false;
+
+	if (!(isar5 | ID_ISAR5_SHA1))
+		return false;
+
+	if (!(isar5 | ID_ISAR5_SHA2))
+		return false;
+
+	if (!(isar5 | ID_ISAR5_CRC32))
```

Comment:
Do we need to care about this one?

Reply:
well, kernel does it also https://github.com/torvalds/linux/blob/master/arch/arm/kernel/setup.c#L480, though CRC32 is not a part of CE

Reply:
@jenswi-linaro so what you prefer: delete it or keep it?